### DISCUSSION TITLE
Bit confused regarding project submission!

### DISCUSSION THREAD

**Discussion by alwinsDen** [wagtail/gsoc#66] [General]
The contributors guide states that I need to submit projects built in Django and Wagtail to my git repo under my name in 2023 folder. So I am just curious what these projects exactly have to be? Are there project topics mentioned anywhere else? or can we build something as per our preference?

**Comment by Nathaniel330** [comment]
Are you talking about the last checklist?  They're tutorials for Django and Wagtail. 
You just have to follow them. Or maybe add new stuff as well.

**Comment by thibaudcolas** [comment]
Hi @alwinsDen, I have updated the guide to make this a bit clearer. It’s indeed the code created during both tutorials we expect people to submit.

This can be directly in this repository, or link us to the GitHub repository where you’ve saved the code for the tutorials.
